docs/quickstart/integrations/aws-lambda/quickstart-guide.mdx

```diff
@@ -348,3 +348,62 @@ handler = Mangum(app)
 
 </BackendTabs.TabItem>
 </BackendTabs>
+
```

Comment:
I noticed that the spacing is wrong in the existing document. Please add a <br /> tag between the previous image and the callout. I'll fix the general spacing next week.
![image](https://github.com/user-attachments/assets/58576b1f-6049-471e-8c6c-fc6d18a06eff)